crypto/src/lib.rs

```diff
@@ -367,10 +380,10 @@ implement_public_crypto_wrapper! {
 /// secret keys.
 ///
 /// ```
-/// use exonum::crypto;
+/// # extern crate exonum_crypto;
```

Comment:
:thinking: Why not `use exonum::crypto`? We reexported crate, shouldn't it work?

Reply:
I've decided not to use reexported `crypto` in examples of `crypto` itself, when we don't use exonum at all.
Why is that bad?

Reply:
Oh, okay, I see it know. I thought they're exonum examples